modules/services/analytics/src/main/java/au/com/shiftyjelly/pocketcasts/analytics/AppLifecycleAnalytics.kt

```diff
@@ -18,18 +18,18 @@ class AppLifecycleAnalytics @Inject constructor(
     /* The date the app was last opened, used for calculating time in app */
     private var applicationOpenedDate: Date? = null
 
-    fun onApplicationInstalledOrUpgraded() {
-        // Track app upgrade and install
-        val versionCode = packageUtil.getVersionCode(appContext)
-        val oldVersionCode = settings.getMigratedVersionCode()
```

Comment:
These are unused in the class constructor now and can be removed:

```
appContext: Context,
settings: Settings,
packageUtil: PackageUtil,
```

Reply:
Thanks for catching that. Fixed in [68daf84](https://github.com/Automattic/pocket-casts-android/pull/1134/commits/68daf84546bb01469e93bf4ea9d5d03122d3c6b6)